src/content/changes/highlights/2021-09-10.md

```diff
@@ -0,0 +1,14 @@
+---
+date: 2021-08-27T10:00:00
+title: Highlights for the week ending September 10, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+[NGINX IC v2.2.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#220---2021-09-09) upgrades to controller container image v1.0.0. **This is a breaking change!** From this version on, only clusters with kubernetes >= 1.19 are supported. Please make sure to read the [upgrading notes](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/README.md#upgrading-notes). ([#218](https://github.com/giantswarm/nginx-ingress-controller-app/pull/218)).
```

Comment:
```suggestion
[NGINX IC v2.2.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#220---2021-09-09) upgrades to upstream v1.0.0. **This is a breaking change!** From this version on, only clusters with kubernetes >= 1.19 are supported. Please make sure to read the [upgrading notes](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/README.md#upgrading-notes). ([#218](https://github.com/giantswarm/nginx-ingress-controller-app/pull/218)).
```

Looks more in line with the below and former announcements.

should we add a note about the breaking change also being that only ingress v1 resources are supported?

Upstream notes are:

```
This release only supports Kubernetes versions >= v1.19. The support for Ingress Object in networking.k8s.io/v1beta is being dropped and manifests should now use networking.k8s.io/v1.
```